Userland/Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -387,6 +388,7 @@ class ComputedValues {
     Vector<ShadowData> const& text_shadow() const { return m_inherited.text_shadow; }
     CSS::Positioning position() const { return m_noninherited.position; }
     CSS::WhiteSpace white_space() const { return m_inherited.white_space; }
+    Optional<CSS::LengthOrCalculated> word_spacing() const { return m_inherited.word_spacing; }
```

Comment:
I gave you bad advice before! `normal` is equivalent to `0`, so this doesn't need `Optional`.

Reply:
I'll remember for next time!